fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/FDBDatabaseRunner.java

```diff
@@ -349,7 +350,8 @@ private RunRetriable(@Nullable List<Object> additionalLogMessageKeyValues) {
                 if (tries + 1 < getMaxAttempts() && retry) {
                     if (LOGGER.isWarnEnabled()) {
                         final KeyValueLogMessage message = KeyValueLogMessage.build("Retrying FDB Exception",
-                                "message", fdbMessage, "code", code);
+                                                                LogMessageKeys.MESSAGE      , fdbMessage,
```

Comment:
As a rule we never try to column-align values like this as it makes editing that much more difficult later. So this should just be:

    LogMessageKeys.MESSAGE, fdbMessage,
    LogMessageKeys.CODE, code

Reply:
DONE
- Removed column alignment.